src/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -263,7 +263,11 @@ Interop.NetSecurityNative.Status status
                     tokenBytes = tokenBytes.Slice(0, length - 1);
                 }
 
+#if NETSTANDARD2_0
+                return Encoding.UTF8.GetString(tokenBytes.ToArray(), 0, tokenBytes.Length);
```

Comment:
@davidsh, is this a perf sensitive code path?  If it is, we could use the unsafe overload of GetString instead, e.g.
```C#
fixed (byte* ptr = tokenBytes)
{
    return Encoding.UTF8.GetString(ptr, tokenBytes.Length);
}
```

Reply:
See https://github.com/dotnet/corefx/pull/42548#discussion_r344981769

Reply:
Ok